GitUI/UserEnvironmentInformation.cs

```diff
@@ -9,18 +9,13 @@ namespace GitUI
     public sealed class UserEnvironmentInformation
     {
         private static bool _alreadySet;
-        private static bool _dirty;
-        private static string _sha;
+        private static bool _dirty = true;
+        private static string _sha = "9999999999999999999999999999999999abcdef";
 
         public static void CopyInformation() => Clipboard.SetText(GetInformation());
 
         public static string GetInformation()
         {
-            if (!_alreadySet)
```

Comment:
Let's keep the exception, but leverage `LicenseManager` and `IsUnitTestActive` constructs for design time and unit tests

https://github.com/gitextensions/gitextensions/blob/99e3c0f52da5cd79598d3ae4e8718390914dae01/GitUI/GitModuleForm.cs#L49-L56

Reply:
So to be clear. You want two sources of info. The design time check info and then the actual info from the info class.   So if the environment info class changes design time will show one thing and runtime another. I totally disagree and think it is foolish.

Reply:
I chose not to do design checking because the control is loading the data from this source.  The source doesn't know about design time.  The source could be used in other places that don't know about design time.

Reply:
Do you see the thought behind it now?  
![image](https://user-images.githubusercontent.com/3629489/47852471-40731080-ddb2-11e8-8b7e-95826c64e507.png)

As soon as you use design time checking you will now have two sources.  The code in UserEnvironmentInfo and then the design time code.  How about we not do that and have ONE source of info and be able to see the info so that when designing we know the form won't clip when resizing.

Reply:
If you ever see all 9's in runtime, then you know an issue has happened.  It's a red herring.

Reply:
Also.  Please try designing FormAbout in master.

Reply:
I altered the content of the source.  Added a new line   
![image](https://user-images.githubusercontent.com/3629489/47853294-a82a5b00-ddb4-11e8-8ae0-211d1769b36c.png)

If we used designtime checking the ui would still look the same in the design checked code because I didn't know there was another place to update the text for design.  With my way, as soon as gitgui is built designer will show the new info as it would come out in runtime.

Reply:
Right.  I need to resize form.  
![image](https://user-images.githubusercontent.com/3629489/47853788-3fdc7900-ddb6-11e8-99d5-367c7bdb3eb5.png)

Reply:
I think I just figured it out. I can have my cake and eat it too. 

```c#
If (design)
{
UserEnvironmentInformation.Initilise(redHerring,true);
}

```

Reply:
Trees meet forest. Ugh. I just didn't see how design checking was going to prevent duplicate sources. I'll get it fixed soon.

Reply:
Trees meet forest. Ugh. I just didn't see how design checking was going to prevent duplicate sources. I'll get it fixed soon.